templates/base/client/.hathora/client.ts.hbs

```diff
@@ -57,31 +60,12 @@ export class HathoraClient {
     token: string,
     stateId: StateId,
     onUpdate?: UpdateCallback,
-    onError?: ErrorCallback
+    onError?: ErrorCallback,
+    transportType: TransportType = TransportType.WebSocket
```

Comment:
it's kind of weird that you can't provide a `transportType` without providing an `onUpdate`. it might be worth moving the last 3 args to an options object instead (will be a small break)